dts/bindings/timer/riscv,mtime-timer.yaml

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
+# SPDX-License-Identifier: Apache-2.0
+
+description: RISC-V Standard mtime Timer.
```

```suggestion
description: RISC-V Machine Timer.
```

The RISC-V spec uses this name to refer to that block, so let's reflect that here as well

---

Done.